execution/executor/src/tests/mock_vm/mock_vm_test.rs

```diff
@@ -42,8 +25,11 @@ fn test_mock_vm_different_senders() {
         txns.push(encode_mint_transaction(gen_address(i), amount));
     }
 
-    let outputs = MockVM
-        .execute_block_no_limit(&into_signature_verified_block(txns.clone()), &MockStateView)
+    let outputs = MockVM::new()
```

Comment:
how come before it didn't require ::new()?

Reply:
`MockVM` is `Self`, and `new()` also returns `Self`, so both approaches work😄 I find having an explicit constructor cleaner and easier to maintain if we change `new()` signature.

Reply:
yeah, 100%.